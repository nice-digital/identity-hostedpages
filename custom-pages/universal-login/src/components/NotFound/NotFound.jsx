import { createBrowserHistory } from "history";

const goBack = (e) => {
  e.preventDefault()
  return createBrowserHistory.goBack();
}

export const NotFound = () => (
  <>
    <h4>Something must have gone slightly wrong!</h4>
    <p><a href="/" onClick={goBack}>&larr; Back</a></p>
  </>
);
