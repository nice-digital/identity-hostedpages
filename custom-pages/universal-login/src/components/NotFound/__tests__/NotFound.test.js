import { render } from "@testing-library/react";
import { NotFound } from "../NotFound";

describe("NotFound components", () => {
  it("should render NotFound correctly", () => {
		const { container } = render(<NotFound />);
    expect(container).toMatchSnapshot();
  })
})
