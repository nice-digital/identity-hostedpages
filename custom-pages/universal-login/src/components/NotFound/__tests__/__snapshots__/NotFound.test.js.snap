// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound components should render NotFound correctly 1`] = `
<div>
  <div
    class="notFound"
  >
    <h4>
      Something must have gone slightly wrong!
    </h4>
  </div>
</div>
`;
